grafonnet/timeseries_panel.libsonnet

```
@@ -0,0 +1,320 @@
+   * @method addOverridesForFieldsOfType(type, overrides) Add a list of overrides for fields of a given type.
+   * @method addOverridesForQuery(queryId, overrides) Add a list of overrides for fields from a given query.
+   */
+  new(
+    title,
+    description=null,
+    graphStyle='line',
+    legendMode='list',
+    legendPlacement='bottom',
+    legendValues=[],
+    lineInterpolation='linear',
+    lineStyle='solid',
+    lineDashSegments=null,
+    lineWidth=1,
+    barAlignment='center',
+    connectNullValues=false,
+    fillOpacity=0,
+    showPoints='auto',
+    pointSize=5,
+    stackSeries='none',
+    gradientMode='none',
+    axisPlacement='auto',
+    axisLabel='',
+    axisWidth='auto',
+    axisSoftMin=null,
+    axisSoftMax=null,
+    axisShowGridLines='auto',
+    axisLogBase=null,
+    repeat=null,
+    repeatDirection='h',
+    repeatMaxPerRow=null,
+    tooltip='single',
+    tooltipSort=null,
+    transparent=null,
+    unit=null,
+    min=null,
+    max=null,
+    decimals=null,
+    seriesName=null,
+    colorMode='palette-classic',
+    fixedColor=null,
+    colorBy='last',
+    noValue=null,
+    thresholdDisplay='off',
+    thresholdMode='absolute',
+  )::
+    {
+      [if description != null then 'description']: description,
+      fieldConfig: {
+        defaults: {
+          color: {
+            mode: colorMode,
+            [if colorMode == 'fixed' && fixedColor != null then 'fixedColor']: fixedColor,
+            [if colorBy != 'last' then 'seriesBy']: colorBy,
+          },
+          custom: {
+            axisLabel: axisLabel,
+            axisPlacement: axisPlacement,
+            [if axisWidth != 'auto' then 'axisWidth']: axisWidth,
+            [if axisSoftMin != null then 'axisSoftMin']: axisSoftMin,
+            [if axisSoftMax != null then 'axisSoftMax']: axisSoftMax,
+            [if axisShowGridLines != 'auto' then 'axisShowGrid']: axisShowGridLines,
+            barAlignment: {
+              left: -1,
+              center: 0,
+              right: 1,
+            }[barAlignment],
+            drawStyle: graphStyle,
+            fillOpacity: fillOpacity,
+            gradientMode: gradientMode,
+            hideFrom: {
+              legend: false,
+              tooltip: false,
+              viz: false,
+            },
+            lineInterpolation: lineInterpolation,
+            [if graphStyle == 'line' then 'lineStyle']: {
+              fill: lineStyle,
+              [if lineStyle == 'dash' then 'dash']: if lineDashSegments != null then lineDashSegments else [10, 10],
+              [if lineStyle == 'dot' then 'dash']: if lineDashSegments != null then lineDashSegments else [0, 10],
+            },
+            lineWidth: lineWidth,
+            pointSize: pointSize,
+            scaleDistribution: {
+              type: if axisLogBase == null then 'linear' else 'logarithmic',
```

I'm running Grafana v9.0.5, where this seemingly should be _log_
```suggestion
              type: if axisLogBase == null then 'linear' else 'log',
```

---

Right you are. Fixed, thanks!